browser/resources/brave_new_tab_page_refresh/components/search/search_results.tsx

```diff
@@ -135,8 +138,10 @@ export function SearchResults(props: Props) {
                 <MatchImage match={match} />
               </span>
               <span className='content'>
-                {match.contents}
-                <span className='description'>{match.description}</span>
+                {mojoString16ToString(match.contents)}
+                <span className='description'>
+                  {mojoString16ToString(match.description)}
+                </span>
```

Comment:
depends on how complicated the implementation of these is, but do we need to cache them

Reply:
Created a new component that takes a `match` prop, similar to `MatchImage`.